### ISSUE TITLE
cms, sitewide [AXE-CORE]: Landmarks must have a unique role or role/label/title (i.e. accessible name) combination

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#4215]
```diff
! Team affected: Facility Locator; Project: LeftNav
```

### [AXE-CORE]: Landmarks must have a unique role or role/label/title (i.e. accessible name) combination

Landmarks must have a unique role or role/label/title (i.e. accessible name) combination. The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable.

Landmark roles are intended to be used sparingly, to identify larger overall sections of the document. Using too many landmark roles can create "noise" in screen readers, making it difficult to understand the overall layout of the page.

Using the `<nav>` element will automatically communicate a section has a role of navigation. If at all possible, prefer using it instead.

If there is more than one navigation landmark role or <nav> element in a document, provide a label for each landmark. This label will allow an assistive technology user to be able to quickly understand the purpose of each landmark.

The `role="navigation"` added to the `div id="vetnav"` conflicts with the `nav` element. 

Recommend using `aria-label` to distinguish between the main and subnav navigation.

## Type of Issue
Template

## Point of Contact

Jennifer

## Acceptance Criteria

As a screenreader user, I want to understand the navigation options so I may clearly navigate the page content.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destinations:
  * https://www.va.gov/pittsburgh-health-care/
  * https://www.va.gov/pittsburgh-health-care/health-services/
  * https://www.va.gov/pittsburgh-health-care/locations/
  * https://www.va.gov/pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive/
  * https://www.va.gov/pittsburgh-health-care/locations/h-john-heinz-iii-department-of-veterans-affairs-medical-center/
  * https://www.va.gov/pittsburgh-health-care/locations/beaver-county-va-clinic/
  * https://www.va.gov/pittsburgh-health-care/locations/belmont-county-va-clinic/
  * https://www.va.gov/pittsburgh-health-care/locations/fayette-county-va-clinic/
  * https://www.va.gov/pittsburgh-health-care/locations/washington-county-va-clinic/
  * https://www.va.gov/pittsburgh-health-care/locations/westmoreland-county-va-clinic/
  * https://www.va.gov/pittsburgh-health-care/events/
  * https://www.va.gov/pittsburgh-health-care/stories/
  * https://www.va.gov/pittsburgh-health-care/news-releases/
  * https://www.va.gov/pittsburgh-health-care/about/
  * https://www.va.gov/pittsburgh-health-care/mission-vision/
  * https://www.va.gov/pittsburgh-health-care/leadership/
  * https://www.va.gov/pittsburgh-health-care/history/
  * https://www.va.gov/pittsburgh-health-care/performance/
  * https://www.va.gov/pittsburgh-health-care/work-with-us/
  * https://www.va.gov/pittsburgh-health-care/work-with-us/jobs-careers/
  * https://www.va.gov/pittsburgh-health-care/work-with-us/internships-fellowships/
  * https://www.va.gov/pittsburgh-health-care/psychology-internships-and-fellowships/
  * https://www.va.gov/pittsburgh-health-care/psychology-internships-and-fellowships/
  * https://www.va.gov/pittsburgh-health-care/work-with-us/volunteer-or-donate/
  * https://www.va.gov/pittsburgh-health-care/programs/
  * https://www.va.gov/pittsburgh-health-care/programs/cardiology/
  * https://www.va.gov/pittsburgh-health-care/research/
  * https://www.va.gov/pittsburgh-health-care/research/safety-security/
  * https://www.va.gov/pittsburgh-health-care/research/research-professionals/
  * https://www.va.gov/pittsburgh-health-care/research/research-professionals/
  * https://www.va.gov/pittsburgh-health-care/contact-us/
  * https://www.va.gov/pittsburgh-health-care/events/mindful-mondays-2/ (and other event detail views)

## Steps to Recreate

1. Enter `https://www.va.gov/pittsburgh-health-care/` in browser
2. Run axe
3. Verify the axe-core error, "Ensures landmarks are unique"
4. Alternatively, start screenreader
5. Verify the conflict in navigation landmarks

### Current Code
**Issue location:**
```html
<div id="vetnav" 
     role="navigation">
```
**Related node:** #va-detailpage-sidebar
```html
<nav id="va-detailpage-sidebar" 
     class="va-c-facility-sidebar usa-width-one-fourth va-sidebarnav">
```
### Recommended Fix
```html
<div id="vetnav" 
     role="navigation" 
     aria-label="main navigation">

<nav id="va-detailpage-sidebar" 
     class="va-c-facility-sidebar usa-width-one-fourth va-sidebarnav" 
     aria-label="secondary navigation">
```
Note: 
Screen readers will announce the type of role the landmark is. Because of this, you do not need to describe what the landmark is in its label. For example, a declaration of `role="navigation"` with an of aria-label="Primary navigation" may be announced redundantly as, "primary navigation navigation".

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 Landmarks must have a unique role or role/label/title (i.e. accessible name) combination](https://dequeuniversity.com/rules/axe/3.4/landmark-unique)
* [MDN ARIA: Navigation Role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role)
* [navigation (role) - Accessible Rich Internet Applications (WAI-ARIA) 1.1 - The definition of 'ARIA Navigtion Role' in that specification.](https://www.w3.org/TR/wai-aria-1.1/#navigation)
* [4.3.6 Navigation - WAI-ARIA Authoring Practices - The definition of 'Navigation Landmark Role' in that specification.](https://www.w3.org/TR/wai-aria-practices-1.1/#aria_lh_navigation)

**Comment by jenstrickland** [comment]
@mmiddaugh No, this is because the main mega nav and the sidebar left nav both are `<nav>`. In the **Possible fix** I provided examples of adding `aria-label="main navigation"` and `aria-label="secondary navigation"` which distinguishes the two areas for screenreaders. 

Ticket #5473 is adding labels for Download links, so that screenreaders have supplementary info.

**Comment by mmiddaugh** [comment]
Thank you for the clarification, @jenstrickland

**Comment by mmiddaugh** [comment]
This issue should be resolved as part of the design changes in [#7212](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7212). Will confirm once changes are available in staging.  Moving this to the Icebox until then.

**Comment by mmiddaugh** [comment]
This is ready for validation, @jenstrickland

**Comment by jenstrickland** [comment]
validated. 🎉
